src/main/content/_assets/js/collections.js

```diff
@@ -1,82 +1,155 @@
+            hideAndEmptyTable();
+            syncColletions(instanceName);
+        }
+    });
+
+    $("#collection-table-body").on("click", ".deactivate-collection-icon", (e) => {
+        let collectionName = e.currentTarget.getAttribute("collection-name");
+        $("#modal-collection-name").text(collectionName);
+    });
+
+    $("#modal-confirm-deactivation").on("click", () => {
+        let collectionName = $("#modal-collection-name").text();
+        deactivateCollection(collectionName);
+    });
+
 });
 
-function handleCollectionsRequests(instanceName){
+function handleInstancesRequests(){
+    hideAndEmptyTable();
+    fetchAllInstances()
+        .then(setInstanceSelections)
+        .then(handleInitialCLIAuth)
+        .then(handleCollectionsRequests);
+        //TODO then select/fetch current instance from url param for the dropdown
+}
+
+function handleCollectionsRequests(instanceName) {
     getCollectionData(instanceName);
     getCliVersion(instanceName);
 }
 
-function getCollectionData(instanceName){
-    if(typeof instanceName === "undefined"){
+function getCollectionData(instanceName) {
+    if (typeof instanceName === "undefined") {
         return;
     }
     return fetch(`/api/auth/kabanero/${instanceName}/collections/list`)
-        .then(function(response) {
+        .then(function (response) {
             return response.json();
         })
         .then(updateCollectionView)
         .catch(error => console.error("Error getting collections", error));
 }
 
-function getCliVersion(instanceName){
-    if(typeof instanceName === "undefined"){
+function getCliVersion(instanceName) {
+    if (typeof instanceName === "undefined") {
         return;
     }
     return fetch(`/api/auth/kabanero/${instanceName}/collections/version`)
-        .then(function(response){
+        .then(function (response) {
             return response.json()
         })
-        .then(setCLIVersion) 
+        .then(setCLIVersion)
         .catch(error => console.error("Error getting CLI Version", error));
 }
 
-function updateCollectionView(collectionJSON){
-    if(typeof collectionJSON === "undefined"){
+function deactivateCollection(collectionName) {
+    let instanceName = $("#instance-accordion").find(".bx--accordion__title").text();
+
+    return fetch(`/api/auth/kabanero/${instanceName}/collections/deactivate/${collectionName}`)
+        .then(function (response) {
+            return response.json()
+        })
+        .then(handleInstancesRequests)
+        .catch(error => console.error(`Error deactivating ${collectionName} collection`, error));
+}
+
+function syncColletions(instanceName) {
+    if (typeof instanceName === "undefined") {
+        return;
+    }
+    return fetch(`/api/auth/kabanero/${instanceName}/collections/sync`)
+        .then(function (response) {
+            return response.json()
+        })
+        .then(handleInstancesRequests)
```

Comment:
should only need to call `handleCollectionsRequests()` you can get the instance name from `let instanceName = $("#instance-accordion").find(".bx--accordion__title").text();`